### ISSUE TITLE
Pandoc addin gives error: Method not found: 'Void MarkdownMonster.mmApp.SetThemeWindowOverride(MahApps.Metro.Controls.MetroWindow, Boolean)'.

### ISSUE THREAD

**Issue by anders-eriksson** [RickStrahl/MarkdownMonster#890]
I have upgraded to Markdown Moster version 2.2.2.1 and added the Pandoc 0.3.6
The adding seems to register correctly, but I get an error dialog when I try to run it

---------------------------
Addin Execution Failed
---------------------------
The 'Pandoc Markdown Parser' addin failed:
Method not found: 'Void MarkdownMonster.mmApp.SetThemeWindowOverride(MahApps.Metro.Controls.MetroWindow, Boolean)'.
You can check to see if there is an update for the Addin available in the Addin Manager. We also recommend you update to the latest version of Markdown Monster.
---------------------------
OK   
---------------------------

Any idea on how to fix this?

From the Error log
2021-12-08 12:28:10 - Addin Pandoc Markdown Parser Execute failed
Method not found: 'Void MarkdownMonster.mmApp.SetThemeWindowOverride(MahApps.Metro.Controls.MetroWindow, Boolean)'.
Markdown Monster v2.2.2.1
10.0.19041.1.amd64fre.vb_release.191206-1406 - en-US - NET 4.8 - 64 bit
sv-SE - en-US    
---
PandocMarkdownParserAddin
   at PanDocMarkdownParserAddin.PandocMarkdownParserWindow..ctor(PanDocMarkdownParserAddin addin)
   at PanDocMarkdownParserAddin.PanDocMarkdownParserAddin.OnExecute(Object sender) in C:\projects\MarkdownMonsterAddins\PanDocMarkdownParserAddin\PanDocMarkdownParserAddin\PanDocMarkdownParserAddin.cs:line 54
   at MarkdownMonster.AddIns.AddinManager.<>c__DisplayClass20_2.<<InitializeAddinsUi>b__0>d.MoveNext() in C:\projects\MarkdownMonsterCode\MarkdownMonster\_Classes\AddInInterfaces\AddinManager.cs:line 149
System.MissingMethodException
---------------------------

**Comment by RickStrahl** [comment]
Make sure you install the latest version of the addin. If you upgraded the old version if already installed won't work as the addin APIs have changed. Update from the Addin Manager.

**Comment by anders-eriksson** [comment]
I uninstalled the addin, using the Addin Manager. Then to be sure I uninstalled Markdown Monster. I alsow removed the PandocAddin folder
Then I installed MM 2.2.2.1 and Pandoc addin via Addin Manager.
**I still get the same error!**

If I look at the properties for the PandocMarkdownParserAdd
![PandocMarkdownParserAddin](https://user-images.githubusercontent.com/17025701/145633809-ee89f948-2dd1-48c5-904a-18d06195a28d.jpg)
in.dll is seem like it is the wrong version?

The version.json says that the version is 0.3.6, but the version in the dll is 0.3.25

The addin is installed in C:\Users\Anders\AppData\Roaming\Markdown Monster\Addins\PandocMarkdownParser
